examples/LoRaWAN/LoRaWAN_End_Device_Reference/LoRaWAN_End_Device_Reference.ino

```diff
@@ -276,21 +234,26 @@ void loop() {
     }
   
   } else if(state == RADIOLIB_ERR_RX_TIMEOUT) {
-    Serial.println(F("timeout!"));
+    // Not really necessary to report normal operation
   
   } else {
     Serial.print(F("failed, code "));
     Serial.println(state);
   }
 
-  // on EEPROM enabled boards, you should save the current session
-  // by calling "saveSession" which allows retrieving the session after reboot or deepsleep
+  // on boards that can save to Flash or EEPROM this saves the session
+  // which allows recall of the session after reboot or deepsleep
   node.saveSession();
 
   // wait before sending another packet
-  uint32_t minimumDelay = 60000;                  // try to send once every minute
-  uint32_t interval = node.timeUntilUplink();     // calculate minimum duty cycle delay (per law!)
-	uint32_t delayMs = max(interval, minimumDelay); // cannot send faster than duty cycle allows
+  uint32_t minimumDelay = 3 * 60 * 1000;          // try to send once every 3 minutes
```

Comment:
Appears I'm not as consistent as I thought with the multiplying out. If it's an uint32_t which is the base for millis() then I'm not aware of any likely issues unless they are hoping to run beyond 49 days. But I haven't traced the call to see if it ends up as int at any point, so I'll check.

Reply:
I think I didn't explain myself fully - `minimumDelay` is not the thing that will overflow here, the overflow might occur during the multiplication. I think the assumption here was that the numbers will get cast to `uint32_t` because of the type of `minimumDelay`. That's not actually true, the constants will remain `int` for the multiplication:

` uint32_t minimumDelay = (int)3 * (int)60 * (int)1000;`

Now, if `int` is only 16 bits wide, it will overflow from 180000 to 48928. You can fix that either by casting the constants, or using suffix, e.g. `1000UL`.